builder.ts

```diff
@@ -85,6 +86,11 @@ export class PackDefinitionBuilder implements BasicPackDefinition {
    * See {@link PackVersionDefinition.networkDomains}.
    */
   networkDomains: string[];
+  /**
+   * See {@link PackVersionDefinition.mcpServers}.
+   * @hidden
+   */
+  mcpServers: MCPServer[];
```

Comment:
I guess it's fine to support multiple MCPs for future compatibility if the runtime supports it, but do we have any known cases of this yet? And without a network domain exception they would all need to be on the same domain?

Reply:
No known cases